client/tools/mgr-cfg/mgr-cfg.spec

```diff
@@ -125,16 +126,24 @@ Obsoletes:      python-%{name} < %{oldversion}
 Provides:       python2-%{oldname} = %{oldversion}
 Obsoletes:      python2-%{oldname} < %{oldversion}
 Requires:       %{name} = %{version}-%{release}
+%if 0%{?suse_version} || 0%{?rhel} >= 8
+Requires:       python2
```

Comment:
On SLE12 this result in an "unresolvable: nothing provides python2".
Just do this %if for `rhel` only. At SUSE we have the definition python == python2 .

Reply:
ok, I think I need more understanding on this as I hit this before.
My understanding (and tests on LEAP 15.2), SUSE detects the required Python version. so if you put
`Requires: python`
it will either resolve to python2 or python3. On my test builds, the package failed because of missing Python 2 (3 was installed instead). This is why I am forcing Python 2 by using Requires: python2.
Or should I also include the actual SUSE version number? (I noticed you wrote SLE12).

Reply:
This would be new to me. In SLE and AFAIK also in Leap python is always python2.

https://build.opensuse.org/package/binary/openSUSE:Leap:15.2/python/standard/x86_64/python-2.7.17-lp152.2.17.x86_64.rpm

The package name in Leap is really only "python". python2 is a "Provides" value. 

Ahh, but our "python3" package "Provides: python = 3.x".

So if you found problems, we can say 

Requires: python < 3

Reply:
THANK YOU! I love your approach. This makes it a lot simpler. I believe I have to revisit one or two of my previous submissions for this.

Reply:
@mcalmer , will the "or" work for SLE12?  If yes, I would do:
`Requires:       (python < 3 or python2)`

Reply:
No, I think it will not work on SLE12.

Reply:
We build this package also for RHEL6 and SLES11. So keep the python2 for rhel >= 8 only, please.